Chocolatey.Cake.Recipe/Content/credentials.cake

```diff
@@ -74,4 +93,13 @@ public static TransifexCredentials GetTransifexCredentials(ICakeContext context)
     return new TransifexCredentials(
         context.EnvironmentVariable(Environment.TransifexApiTokenVariable)
     );
+}
+
+public static DockerSourceCredentials GetDockerSourceCredentials(ICakeContext context)
+{
+    return new DockerSourceCredentials(
+        context.EnvironmentVariable("DOCKER_USER"),
+        context.EnvironmentVariable("DOCKER_PASSWORD"),
+        context.EnvironmentVariable("DOCKER_SERVER")
```

Comment:
Do we want/need to be able to control the names of these environment variables, in the same way that you did with the SonarQube PR?